### PR TITLE
First release for X-NUCLEO-IKS01A1 library

### PR DESCRIPTION
**Pull request by cparata**
I created a dedicated library for X-NUCLEO-IKS01A1 in order to group all the examples for this board just in one package. If you want, we can remove the X-NUCLEO-IKS01A1 examples that are in LSM6DS0, LSM6DS3, HTS221, LPS25HB and LIS3MDL libraries because they are already included in this package.

### CODE REVIEW

File: library.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name=STM32duino X-NUCLEO-IKS01A1
+version=1.0.0
+author=STMicroelectronics
+maintainer=stm32duino
+sentence=Allows controlling the ST X-NUCLEO-IKS01A1 expansion board
+paragraph=This library provides the drivers and several sample applications to control ST X-NUCLEO-IKS01A1 expansion board
```
Comment (fpistm) [inline library.properties]: It provides only examples no?

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+* X_NUCLEO_IKS01A1_DoubleTap: This application shows how to detect the double tap event using the X-NUCLEO-IKS01A2 (only with LSM6DS3).
+* X_NUCLEO_IKS01A1_FreeFall: This application shows how to detect the free fall event using the X-NUCLEO-IKS01A2 (only with LSM6DS3).
+* X_NUCLEO_IKS01A1_MultiEvent: This application shows how to detect free fall, tap, double tap, tilt, wake up,
+6D Orientation and step events using the X-NUCLEO-IKS01A2 (only with LSM6DS3).
+* X_NUCLEO_IKS01A1_Pedometer: This application shows how to use X-NUCLEO-IKS01A2 to count steps (only with LSM6DS3).
+* X_NUCLEO_IKS01A1_Tap: This application shows how to detect the single tap event using the X-NUCLEO-IKS01A2 (only with LSM6DS3).
+* X_NUCLEO_IKS01A1_Tilt: This application shows how to detect the tilt event using the X-NUCLEO-IKS01A2 (only with LSM6DS3).
+* X_NUCLEO_IKS01A1_WakeUp: This application shows how to detect the wake-up event using the X-NUCLEO-IKS01A2 (only with LSM6DS3).
+
+# Notes
```
Comment (fpistm) [inline README.md]: Same comment than the  X-NUCLEO-IKS01A2.
Maybe name this section dependencies and provide link to stm32duino organisation

**Review by fpistm — APPROVED** [review]
only 2 questions else ok

File: library.properties (hunk 1 above)
Comment (cparata) [reply library.properties]: yes right! I will update it

**Review by fpistm — APPROVED** [review]

**Pull request merged**